packages/blocks/azure/src/lib/actions/azure-advisor-action.ts

```diff
@@ -0,0 +1,87 @@
+import { createAction, Property } from '@openops/blocks-framework';
+import { azureAuth, dryRunCheckBox } from '@openops/common';
+import { logger } from '@openops/server-shared';
+import { runCommand } from '../azure-cli';
+import { subDropdown, useHostSession } from '../common-properties';
+
+export const advisorAction = createAction({
+  auth: azureAuth,
+  name: 'advisor',
+  description: 'Get Azure Advisor Cost Recommendations',
+  displayName: 'Get Advisor Cost Recommendations',
+  props: {
+    useHostSession: useHostSession,
+    subscriptions: subDropdown,
+    ids: Property.Array({
+      displayName: 'Resource IDs',
+      description:
```

Comment:
copied the description from azure directly

Reply:
I have to say I don't get it.

Reply:
yea i think what they mean is no other resource related arguments should be filled out. 

Maybe "If provided, cannot also provide 'resource group' " and the same for resource group?

Reply:
It is mutually exclusive, but i fear a toggle might be a bit confusing no? Wed still have to add a description of why we toggle it

Reply:
Hmm, I imagined something like "Filter by Resource IDs" vs "Filter by Resource Group".

Reply:
both can be empty yes